server/domain/devices/postgre.go

```diff
@@ -12,12 +12,12 @@ import (
 const (
 	host     = "localhost"
 	port     = "5432"
-	user     = "school"
-	password = "eliftech"
+	user     = "max"
```

Comment:
Merging this would break everybody's applications, please change your DB credentials (through `psql` probably) or write here some code to get these values from shell environment